### DISCUSSION TITLE
Allow 1 author to be in multiple author categories

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Authors#1875] [Feature Requests]
![image](https://github.com/user-attachments/assets/3ea0f569-261f-4d1f-9d8a-0db2e5c07cd9)

You can see that I operate with several "Categories" ("Forfattere" [authors], "Oversættere" [translators], "Grafikere" [designers], "Illustratorer" [illustrator] etc.)

Right now a user/author can only figure in ONE of these categories on a post - but I need them to figure in several categories on the same post. (As the same person can both be the Author and the Illustrator, or be the Illustrator and the Designer on the same book).

request: https://secure.helpscout.net/conversation/2650525566/224207?viewId=6932143

**Comment by andrewmunroe** [comment]
I would like to see this feature as well! Our current workaround is to create combined categories as needed e.g. 'Writer/Photographer'
